src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -439,6 +414,7 @@ internal static int PathStartSkip(string path)
         /// <summary>
         /// True if the given character is a directory separator.
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I'm very surprised this isn't being inlined.  Are you sure?  Into which call site?